win64.mak

```diff
@@ -506,4 +506,9 @@ install: phobos.zip
 
 auto-tester-build: targets
 
-auto-tester-test: unittest
+JOBS=4
+GMAKE=gmake
+
+auto-tester-test:
```

Comment:
This can be quite dangerous as it might result in the normal unittest command no longer working
This I highly recommend remove the no longer tested code.
GNUmake is required for dmd already anyhow and we have approval from A&W for getting rid of the DigitalMars make Makefiles.

Reply:
This is currently meant as an experiment to speed up the build server.

> we have approval from A&W for getting rid of the DigitalMars make Makefiles.

Good, I didn't know that.

So far gmake is only necessary for testing. I think these changes also allow building the library, but invoking the druntime build doesn't yet work.